src/app/study/[id]/focus/_components/Time.tsx

```
@@ -0,0 +1,33 @@
+export default function Time({
+  isActive,
+  minutes,
+  seconds,
+  timerColor,
+}: {
+  isActive: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)